[Data Release] Improvements (#3062)
aces/Loris: modules/data_release/test/data_release_test_plan.md

# Data Release Module Test Plan 

> Make sure that regular users can see the data release module without the 

Reviewer: ```suggestion
Ensure that regular users can see the data release module without the 
```

> 'Upload File' and 'Add Permission' buttons at the top

Reviewer: ```suggestion
'Upload File' and 'Add Permission' buttons at the top.
```

## Upload File
1. Give the user the `data_release_upload` *(Data Release: Upload file)* permission. 

>  Make sure the user can upload new files to the module

Reviewer: ```suggestion
 Ensure the user can upload new files to the module.
```

 
2. As the 'superuser', click on the 'Upload File' button to upload a file 

>  and make sure the uploaded file is available in the list of files

Reviewer: ```suggestion
 and ensure the uploaded file is available in the list of files.
```

 
## Permissions

1. Give the user the `data_release_edit_permissions` *(Data Release: Give user 

> permission to view files)* permission

Reviewer: ```suggestion
permission to view files)* permission.
```

> 2. Click on the 'Add Permission' button

Reviewer: ```suggestion
2. Click on the 'Add Permission' button.
```

3. Select another 'User ID' (that you have access to and is not a 

>  'superuser')

Reviewer: ```suggestion
 'superuser').
```

 
4. Select a 'Data Release ID' (ideally a file that the 'User ID' has no 

>  access to yet) (note that at least one file needs to be uploaded in the 

Reviewer: ```suggestion
 access to yet). Note that at least one file needs to be uploaded in the 
```

 module to be able to set permissions to access a file, otherwise, the drop 

>  down for 'Data Release ID' will remain empty)

Reviewer: ```suggestion
 down for 'Data Release ID' will remain empty.
```

> 5. Make sure that this 'User ID' can now see the file now that the 

Reviewer: ```suggestion
5. Ensure that this 'User ID' can now see the file now that the 
```

>  permission was added for him/her

Reviewer: ```suggestion
 permission was added for him/her.
```

## Superuser

> Make sure that a 'superuser' can see the 'Upload File' and 'Add 

Reviewer: ```suggestion
Ensure that a 'superuser' can see the 'Upload File' and 'Add 
```

Permission' buttons at the top of the module, as well as the list of already 

> uploaded files

Reviewer: ```suggestion
uploaded files.
```

## Sort table
 

> Make sure that clicking on the table headers will sort the data present 

Reviewer: ```suggestion
Ensure that clicking on the table headers will sort the data present 
```

>  in the data release table according to that column\

Reviewer: ```suggestion
 in the data release table according to that column.
```
